### DISCUSSION TITLE
Discontinued?

### DISCUSSION THREAD

**Discussion by nixahnung** [patrickpr/trapdirector#73] [General]
Hi,

I just installed the module and I find it very useful. I had to adjust some php to make it workable on Ubuntu 22.04 (php8.1, etc.).

Hence the question: Has this project been discontinued?

That would be a shame, because it fits well into the monitoring world.

Translated with Google. sorry

**Comment by Tqnsls** [comment]
if that were the case I would love if icinga would take over the development for this project.

**Comment by MagnaFrisia90** [comment]
@nixahnung Did you also experience #79 ?
